package/rebuild-todo

```diff
@@ -219,6 +224,22 @@ read <&1
 
 pkgctl repo clone "${packages[@]}"
 
+if ((IMPORT_KEYS)); then
+	echo -e "Importing PGP keys...\n"
+	# Only add paths that actually have key(s) to import, ignore errors and don't exit on paths that don't
+	key_paths=$(find "${packages[@]/%//keys/pgp}" -type f 2>/dev/null || true)
+
+	if [[ -n "$key_paths" ]]; then
+		if [[ -z $OFFLOAD ]]; then
```

Comment:
Please make the quoting consistent here 😊

Reply:
Indeed!